assets/icons/logo.svg

```diff
@@ -1,9 +1,156 @@
-<?xml version="1.0" encoding="utf-8"?>
-<svg viewBox="51.683 53.059 323.109 369.725" xmlns="http://www.w3.org/2000/svg">
-  <path d="M 369.724 141.841 L 219.126 54.494 C 215.922 52.58 211.924 52.58 208.72 54.494 L 56.751 141.841 C 53.589 143.676 51.643 147.071 51.684 150.742 L 51.684 325.027 C 51.643 328.682 53.589 332.091 56.751 333.926 L 208.036 421.273 C 211.211 423.287 215.264 423.287 218.442 421.273 L 369.724 333.515 C 372.903 331.679 374.844 328.269 374.791 324.615 L 374.791 150.33 C 374.694 146.811 372.779 143.594 369.724 141.841 Z" style="fill: rgb(245, 168, 0);"/>
-  <path d="M 287.301 288.613 C 274.591 288.613 261.883 284.791 250.727 277.878 C 228.417 291.785 199.892 291.785 177.584 277.878 C 166.427 284.791 153.723 288.613 141.014 288.613 L 122.728 288.613 L 122.728 304.88 L 141.014 304.88 C 153.629 304.88 166.062 302.033 177.584 296.83 C 200.623 307.317 227.688 307.317 250.727 296.83 C 262.246 302.113 274.684 304.88 287.301 304.88 L 305.588 304.88 L 305.588 288.613 L 287.301 288.613 Z M 140.554 272.349 L 141.014 272.349 C 155.644 272.349 168.625 265.191 177.584 256.083 C 186.545 265.191 199.526 272.349 214.154 272.349 C 228.783 272.349 241.768 265.191 250.727 256.083 C 259.688 265.191 272.672 272.349 287.301 272.349 L 287.755 272.349 L 305.035 218.017 C 305.765 215.906 305.588 213.629 304.489 211.675 C 303.39 209.726 301.378 208.26 299 207.608 L 287.301 204.193 L 287.301 166.618 C 287.301 157.674 279.07 150.354 269.012 150.354 L 241.584 150.354 L 241.584 125.954 L 186.728 125.954 L 186.728 150.354 L 159.298 150.354 C 149.241 150.354 141.014 157.674 141.014 166.618 L 141.014 204.193 L 129.218 207.608 C 126.84 208.26 124.829 209.726 123.734 211.675 C 122.635 213.629 122.361 215.906 123.182 218.017 L 140.554 272.349 Z M 159.298 166.618 L 269.012 166.618 L 269.012 198.909 L 214.154 182.886 L 159.298 198.909 L 159.298 166.618 Z" style="fill: rgb(255, 255, 255);"/>
-  <line style="stroke-linecap: round; fill: rgb(255, 255, 255); stroke: rgb(255, 255, 255); stroke-width: 19px; stroke-dasharray: 30;" x1="137.797" y1="330.007" x2="295.241" y2="330.007"/>
-  <g transform="matrix(6.135525, 0, 0, 6.135525, 175.776047, 233.333344)" style="">
-    <rect fill="none" height="24" width="24"/>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Will this SVG directly go to the webpage? If yes, then we should run it through some minimizer first.

Reply:
It's currently imported via an <img> tag. I can try in-lining the SVG and use Hugo's built in minifier.

Reply:
Just noticed you minified it. Looks better now.